examples/stmicro/stm32/src/stm32f1xx/uart.zig

```diff
@@ -0,0 +1,38 @@
+const std = @import("std");
+const microzig = @import("microzig");
+
+const RCC = microzig.chip.peripherals.RCC;
+const stm32 = microzig.hal;
+
+const usart2: stm32.uart.UART = @enumFromInt(0);
+const gpio = stm32.gpio;
+const TX = gpio.Pin.from_port(.A, 9);
+const RX = gpio.Pin.from_port(.A, 10);
+
+pub fn main() !void {
```

Comment:
Let's name this one _uart_echo.zig_ and make it behave more similar to the rp2xxx version